aea/package_manager/v1.py

```diff
@@ -219,6 +218,11 @@ def update_package_hashes(self) -> "PackageManagerV1":
 
         return self
 
+    @staticmethod
+    def _calculate_hash(package_path: str) -> str:
+        """Calculate hash for path."""
+        return IPFSHashOnly.get(str(package_path))
```

Comment:
not worth a method, please don't because this is all interface we will have to preserve into the future for reasons of compatibility :cry:

Reply:
@Karrenbelt 
actually i like such kind of utlility methods that interacts with some external things

in case you want to change hash calc tool/algo, just override the method without looking over the code.
so this makes sense for me

Reply:
and easy to mock)

Reply:
Either `package_path`'s type hint is incorrect or `str(package_path)` is redundant.

Reply:
> @Karrenbelt actually i like such kind of utlility methods that interacts with some external things
> 
> in case you want to change hash calc tool/algo, just override the method without looking over the code. so this makes sense for me

ok I don't like to expose any more interface than is absolutely necessary. 1. Interface that we now need to maintain, and 2. it adds another alias for an existing method (which is the lesser of these two evils).

Let me recommend to do it this way then:
```
class PackageManagerV1(BasePackageManager):
    """Package manager V1"""
    
    get_ipfs_hash = IPFSHashOnly.get
    
```

Reply:
fixed

Reply:
@Karrenbelt 
tried your variant
```
aea/package_manager/v1.py:237: error: Invalid self argument "PackageManagerV1" to attribute function "_get_ipfs_hash" with type "Callable[[str, bool, bool], str]"
aea/package_manager/v1.py:237: error: Argument 1 has incompatible type "str"; expected "bool"
```

dont want to investigate why it does not work for mypy, tests are ok.

so left it as is.

actually good point to discuss.